stats/kafka/utils.go

```diff
@@ -0,0 +1,38 @@
+package kafka
+
+import (
+	"encoding/json"
+
+	"github.com/loadimpact/k6/stats"
+	"github.com/loadimpact/k6/stats/influxdb"
+	jsonCollector "github.com/loadimpact/k6/stats/json"
+)
+
+func formatSamples(format string, samples []stats.Sample) ([]string, error) {
+	var metrics []string
+
+	switch format {
+	case "influx":
+		i, err := influxdb.New(influxdb.Config{})
```

Comment:
I'd be super cool if we can honour the "[InfluxDB tags as fields setting](https://github.com/loadimpact/k6/blob/master/release%20notes/upcoming.md#influxdb-collector-add-support-for-sending-certain-sample-tags-as-fields-585)" ([PR](https://github.com/loadimpact/k6/pull/585)) here as well. 

It should be possible if you make `formatSamples()` a method of `kafka.Collector` and somehow pass to the constructor the whole `Collectors` config struct and save it.

Reply:
Hmm but then [this](https://github.com/loadimpact/k6/issues/620) would become even more complicated... Not sure how much we can abuse the `strvals` library and if we can encode the influxdb config in it as well.

Reply:
@na-- So I think I was able to get this to work locally ... but because of how the influx collector parses the args you need to provide it a fake config string to have a query parameter for tags ... e.g.

```
brokers={broker2,broker3:9092},topic=someTopic2,format=influxdb,influxdb=http://influx:20?tagsAsFields=vu
```

So even though I'm not really sending to an influxdb instance I need to specify a "url" (`influxdb=http://influx:20?tagsAsFields=vu`) to pass the parameter, even though it's not an actual influx db instance parameter and is meant actually for k6.

I can still proceed with this ... but maybe an issue on how to parse influxdb config should be created and addressed first?

Reply:
Hmm I'd be okay if we support this but require that a dummy url is specified at this time, and then in the future fix it properly as part of the changes needed for https://github.com/loadimpact/k6/issues/586.